### PR TITLE
fix:Text Colour is visible inside the button

### PR DESCRIPTION
**Pull request by ivary43**
Fixes #{417}

Please Add Screenshots If any UI changes.

screenshot_20170927-001021

- \[x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- \[x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- \[x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/layout/fragment_third_party_transfer.xml [hunk 1]
```diff
@@ -241,6 +241,8 @@
                         android:layout_marginTop="10dp"
                         android:layout_weight="0.5"
                         app:backgroundTint="@color/white"
+                        android:textColor="@color/primary"
+                        style="@style/Widget.AppCompat.Button.Borderless"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/fragment_third_party_transfer.xml:245]: @ivary43 Remove this line, we want a border.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
@ivary43, I want one more change here, Please replace the button text `Review Transfer` to `Review` only.

File: app/src/main/res/layout/fragment_third_party_transfer.xml (hunk 1 above)
Comment (ivary43) [reply app/src/main/res/layout/fragment_third_party_transfer.xml:245]: Okay i will do it .one thing more the cancel button is not working .

**Comment by ivary43** [comment]
@therajanmaurya  there are two pages with "review transfer " text ,should i change both.

**Comment by therajanmaurya** [comment]
@ivary43 which those pages?

**Comment by ivary43** [comment]
Transfer and third party transfer .

**Comment by dilpreet2028** [comment]
@ivary43 when you will change the value for `review_transfer` in strings.xml as `Review`, it will change for both of them. Go ahead make the change.

**Comment by therajanmaurya** [comment]
@ivary43 Make the change and let me know.

**Comment by ivary43** [comment]
@therajanmaurya @dilpreet96 I am unable to log in in the app by given  demo credentials  .What should I do.A empty toast is popping up .
screenshot_20171001-220505

**Comment by therajanmaurya** [comment]
Please pull again and build, I have fixed it.

**Pull request closed without merging**